go.mod

```diff
@@ -42,10 +42,16 @@ replace (
 	github.com/spf13/cobra => github.com/chmouel/cobra v0.0.0-20200107083527-379e7a80af0c
 )
 
-// Pin k8s deps to 0.17.6
+// TODO(mattmoor): DO NOT SUBMIT
```

Comment:
WIP /hold

Reply:
FWIW, we have a Github action that will flag `DO NOT SUBMIT` lines and add a failing check to the PR.

Happy to send y'all a PR with that if you have any interest?  @vdemeester @ImJasonH @bobcatfish @afrittoli